examples/parameter_replacement/parameter_replacement.ipynb

```diff
@@ -0,0 +1,316 @@
+   "source": [
+    "So we can compare them key-by-key. Here's Parsley:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pars_sys.handlers[\"Angles\"].potentials"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And Sage:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sage_sys.handlers[\"Angles\"].potentials"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here's the relevant bits next to each other. See if you can find them in the above!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for potential_key in pars_sys.handlers[\"Angles\"].potentials:\n",
+    "    print(potential_key.id)\n",
+    "    for ff, sys in [(\"Sage\", sage_sys), (\"Parsley\", pars_sys)]:\n",
+    "        print(f\"    {ff}\")\n",
+    "        for k, v in sys.handlers[\"Angles\"].potentials[potential_key].parameters.items():\n",
+    "            print(f\"        {k}: {v}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Updating Parsley with Sage angles"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-danger\" style=\"max-width: 700px; margin-left: auto; margin-right: auto;\">\n",
+    "    <b>🛑 For Educational Purposes Only</b><br />\n",
+    "    Don't combine parameters from different force fields or \"update\" one part of a force field without careful consideration. Parameters are designed and optimized to work within a particular context, and moving them to a completely different force field is usually a bad idea.\n",
+    "</div>\n",
+    "\n",
+    "Just to demonstrate changing parameters, lets \"update\" our Parsley Interchange to use Sage angle parameters. This will definitely hurt force field performance, as each set of angle parameters are optimized for different non-bonded parameters, but it demonstrates the API, and we won't be running any simulations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Generally, updating a `PotentialHandler` involves updating both dictionaries, `slot_map` and `potentials`. Since both of our Interchanges use the same SMIRKS codes and parameter types, we could skip updating the `slot_map`, but it's best practice to do both steps every time. It's good practice to clear the old values so that you don't accidentally wind up with redundant parameters (unless that's what you want), so let's start with that:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pars_sys.handlers[\"Angles\"].slot_map.clear()\n",
+    "pars_sys.handlers[\"Angles\"].potentials.clear()"
```

Comment:
I thought it would be good to show users how to clear old parameter handlers, in case someone tries to add a parameter handler with different SMIRKS covering the same chemistry. This is what I came up with, is there a better way?